src/plugins/intel_cpu/src/nodes/mkldnn_reduce_node.cpp

```diff
@@ -70,6 +71,29 @@ using namespace Xbyak;
 #define GET_PTR_NCD_BASE_PTR_N_BLK const uint8_t    *in_ptr_ncd    = in_ptr_n     + src_data_size * (icb * ID + id) * IH * IW * blk_size; \
                                          uint8_t    *out_ptr_ncd   = out_ptr_n    + dst_data_size * (ocb * OD + od) * OH * OW * blk_size;
 
+namespace {
+struct ReduceKey {
+    mkldnn::primitive_attr attr;
+
+    size_t hash() const;
+    bool operator==(const ReduceKey& rhs) const;
```

Comment:
I bet we need more elements in key, at least different values in jit_config_params will generate different kernels.
```
struct jit_reduce_config_params {
    ReduceLayoutType layout;
    Algorithm reduce_mode;
    mkldnn::memory::data_type src_dt;
    mkldnn::memory::data_type dst_dt;
    int src_data_size;
    int dst_data_size;
};
```
https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/mkldnn_reduce_node.h#L21-L28

Reply:
Done. Thanks Chenhu!